### PR TITLE
Unit tests for the actual mysql driver

### PR DESCRIPTION
**Pull request by sushshring**

### CODE REVIEW

**Comment by stan29308** [comment]
It looks like Travis failed while trying to locate "@fluffy-spoon/substitute" which you also updated in the package.json file. I'm not sure how that happened because Travis did npm install.

File: test/services/database/svc/mysql-uow.service.test.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
+import { Arg, Substitute } from '@fluffy-spoon/substitute';
```
Comment (stan29308) [inline test/services/database/svc/mysql-uow.service.test.ts]: Is Arg used anywhere?

**Comment by sushshring** [comment]
Yeah I’m not sure why Travis is having a stronk. I updated the library to a new version but it’s up on npmjs and everything. Might just not have propagated. I’ll rerun the build.

File: test/services/database/svc/mysql-uow.service.test.ts (hunk 1 above)
Comment (sushshring) [reply test/services/database/svc/mysql-uow.service.test.ts]: Nope. Good catch.

**Pull request merged**